spec/models/room_spec.rb

```diff
@@ -3,5 +3,6 @@
 RSpec.describe Room, type: :model do
   describe "relationships" do
     it { should belong_to :hotel }
+    it { should have_many :guests }
```

Comment:
There's one more association/relationship you're missing; `should have_many :guest_rooms` so that your line 6 looks like `it { should have_many(:guests).through(:guest_rooms) } `.